planet/order_request.py

```diff
@@ -108,18 +108,18 @@ def product(item_ids: List[str],
             are not valid bundles or if item_type is not valid for the given
             bundle or fallback bundle.
     '''
-    product_bundle = specs.validate_bundle(product_bundle)
-    item_type = specs.validate_item_type(item_type, product_bundle)
+    validated_product_bundle = specs.validate_bundle(product_bundle)
+    item_type = specs.validate_item_type(item_type, validated_product_bundle)
 
     if fallback_bundle is not None:
-        fallback_bundle = specs.validate_bundle(fallback_bundle)
-        specs.validate_item_type(item_type, fallback_bundle)
-        product_bundle = ','.join([product_bundle, fallback_bundle])
+        validated_fallback_bundle = specs.validate_bundle(fallback_bundle)
+        specs.validate_item_type(item_type, validated_fallback_bundle)
+        validated_product_bundle = ','.join([validated_product_bundle, validated_fallback_bundle])
```

Comment:
Here, though, mypy's analyzer can't find that anything on the right side will be None, so it doesn't care if we reassign.